tests/run_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+set -u 
+set -o pipefail
+
+## run setup
+python setup.py install
```

Comment:
It's best to put this in the github actions file, and keep this file as strictly running the package. Installing it here would be a very unexpected side effect

Reply:
moved install line to gh-actions file as suggested